### PR TITLE
docs: add yubikey setup instructions

### PR DESCRIPTION
**Pull request by mrjoelkamp**
## Summary
* Adds more detailed instructions on how to configure YubiKey PIV using the YubiKey Manager UI

### CODE REVIEW

File: docs/YUBIKEY-PIV-SETUP.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+1. Select `Change PUK`
+
+#### Set Management Key
+
+The management key is used to perform many YubiKey management operations, such as generating a key pair.
+
+1. Navigate to the `Applications` > `PIV` menu in the YubiKey Manager UI
+1. Under `PIN Management`, select `Configure PINs`
+1. Under `Management Key`, select `Change Management Key`
+1. Select the algorithm, recommend selecting `AES256`
```
Comment (joostd) [inline docs/YUBIKEY-PIV-SETUP.md:56]: The option to use an AES key is only available for YubiKeys with firmware 5.4.2 or newer. See https://docs.yubico.com/hardware/yubikey/yk-tech-manual/yk5-overview.html#piv-management-key-aes
On older firmware, a TDES management key is used.

File: docs/YUBIKEY-PIV-SETUP.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+A new YubiKey is configured with a default PIN, PUK (PIN unlock code), and Management Key.
+
+The default PIN codes must be updated with new values that you remember or store securely.
+
+PIN codes are used for signing operations and to unlock a device.
+
+#### Reset PIV to Defaults
+
+1. Navigate to the `Applications` > `PIV` menu in the YubiKey Manager UI
+1. Under `Reset`, select `Reset PIV`
```
Comment (joostd) [inline docs/YUBIKEY-PIV-SETUP.md:25]: It is probably obvious, but users should be aware that all existing PIV data will be destroyed.

**Review by joostd — COMMENTED** [review]
LGTM - 2 minor comments

**Review by jku — APPROVED** [review]
Thanks, this looks good. 

I agree with joostd though: _"Reset PIV to defaults"_ maybe should mention that this will destroy any PIV keys that have been created before.

File: docs/YUBIKEY-PIV-SETUP.md (hunk 2 above)
Comment (mrjoelkamp) [reply docs/YUBIKEY-PIV-SETUP.md:30]: added https://github.com/theupdateframework/tuf-on-ci/pull/229/commits/491bc51be16e05bb9fbdca473b48741157f6e308

File: docs/YUBIKEY-PIV-SETUP.md (hunk 1 above)
Comment (mrjoelkamp) [reply docs/YUBIKEY-PIV-SETUP.md]: thanks! https://github.com/theupdateframework/tuf-on-ci/pull/229/commits/d1ef03d927b6eb2c44492d21fd2e1884034243ba

**Comment by mrjoelkamp** [comment]
> Thanks, this looks good.
> 
> I agree with joostd though: _"Reset PIV to defaults"_ maybe should mention that this will destroy any PIV keys that have been created before.

Yes, I added a couple more alerts following https://github.com/orgs/community/discussions/16925 format

**Comment by jku** [comment]
thanks!

**Pull request merged**